src/index.js

```diff
@@ -54,9 +54,29 @@ const singleUserHandler = async (
   req,
   type,
   replyToken,
+  timeout,
   userId,
   otherFields
 ) => {
+  // reply before timeout
+  // the reply token becomes invalid after a certain period of time
+  // https://developers.line.biz/en/reference/messaging-api/#send-reply-message
+  let isReplied = false;
+  setTimeout(function() {
+    if (isReplied) return;
+
+    isReplied = true;
```

Comment:
We can add
```
console.log(`[LOG] Timeout ${....}`)
```

And we analyze what kind of message causes timeout next week, by look them up in logentries.